.vscode/settings.json

```diff
@@ -0,0 +1,4 @@
+{
+  "swiftformat.enable": false,
+  "files.trimTrailingWhitespace": true
+}
```

Comment:
oops.  :)

Reply:
Gah. Oops

Reply:
On that note, why doesn’t this repo use `swiftformat`?

Reply:
Which _swiftformat_, the new one in the Apple repo?  There isn't yet an general style guide the community has adopted, so we haven't bothered to chase one yet (just like different parts of Swift have different formats at the moment).

Reply:
Great question. We’ve been using the non-Apple `swiftformat`, not the new `swift-format`. A blessed standard format is one of the best aspects of working with Go.